src/guides/v2.3/release-notes/release-notes-2-3-5-open-source.md

```diff
@@ -269,15 +278,15 @@ We have fixed hundreds of issues in the Magento 2.3.5 core code.
 
 <!--- MC-30067-->
 
-*  You can now assign a default watermark to a theme. Previously, after assigning the watermark, Magento threw an fatal error.
+*  You can now assign a default watermark to a theme. Previously, after assigning the watermark, Magento threw a fatal error.
 
 <!--- MC-29876-->
 
 *  Magento now displays product images in the mini cart without distortion. Previously, Magento stretched the image in the mini cart to fill the entire width and height of the image container.
 
 <!--- MC-29652-->
 
-*  The Recently View Products feature now shows products associated only with the current store view in multi-store deployments when **Stores** > **Configurations** > **Catalog** > **Recently Viewed/Compared Products** > **Show for Current** is set to **store view**. Previously, Magento displayed recently viewed products from all websites, no matter which website the product was assigned to.
+*  The Recently View Products feature now shows products associated only with the current store view in multi-store deployments when **Stores** > **Configurations** > **Catalog** > **Recently Viewed/Compared Products** > **Show for Current**  is set to **store view**. Previously, Magento displayed recently viewed products from all websites, no matter which website the product was assigned to.
```

Comment:
Should probably be: The Recently View**ed** Products feature

Reply:
changed in source -- thanks